.travis.yml

```diff
@@ -16,6 +16,8 @@ jobs:
         - npm run unit
 
     - os: "windows"
+      name: 'Unit tests: Windows/Chromium'
+      node_js: "10.19.0"
```

Comment:
This seems duplicated:

```suggestion
```

Reply:
Gotta put it twice to be _extra sure_ on the version 🤦